src/encoded/static/components/genome_browser.js

```diff
@@ -108,9 +108,9 @@ const getDefaultCoordinates = (assembly, annotation, ignoreCache = false) => {
 
     if (gVState && gVAssembly === assembly && gVAnnotation === annotation) {
         const savedState = gVState ? JSON.parse(gVState) : {};
-        ({ contig, x0, x1 } = savedState);
-    }
-    if (assembly === 'GRCh38') {
+        ({ contig } = savedState);
+        ({ x0, x1, pinnedFiles } = savedState);
```

Comment:
You are not using `pinnedFiles`. It gets overwritten every time.

Reply:
This s major. The other comments are minor/insignificant.

Reply:
which line are they getting overwritten?

Reply:
Never mind, I get it now.